x-pack/plugins/fleet/server/services/setup.ts

```diff
@@ -123,14 +124,26 @@ async function createSetupSideEffects(
   return { isIntialized: true };
 }
 
-export async function setupFleet(
-  soClient: SavedObjectsClientContract,
-  callCluster: CallESAsCurrentUser,
-  options?: { forceRecreate?: boolean }
-) {
-  // Create fleet_enroll role
-  // This should be done directly in ES at some point
-  const res = await callCluster('transport.request', {
+async function updateFleetRole(callCluster: CallESAsCurrentUser) {
```

Comment:
`updateFleetRoleIfExists` so we can remove the comment below?

Reply:
good call, done